projects/native/App.js

```diff
@@ -1,63 +1,63 @@
-import React from 'react';
-import { Platform, StatusBar, StyleSheet, View } from 'react-native';
-import { AppLoading, Asset, Font, Icon } from 'expo';
-import AppNavigator from './navigation/AppNavigator';
-const navigationPersistenceKey = __DEV__ ? 'NavigationStateDEV' : null;
+import React from "react";
+import { Platform, StatusBar, StyleSheet, View } from "react-native";
+import { AppLoading, Asset, Font, Icon } from "expo";
```

Comment:
you'll probably wanna get rid of these expo imports.

`Font` and `Asset` sound are part of [unimodules](https://github.com/unimodules/react-native-unimodules) and sound like something worth exploring in the future - but we don't have a need for them or what they do right now